eth/filters/filter_system_test.go

```diff
@@ -680,3 +685,73 @@ func flattenLogs(pl [][]*types.Log) []*types.Log {
 	}
 	return logs
 }
+
+func TestVoteSubscription(t *testing.T) {
+	t.Parallel()
+
+	var (
+		db      = rawdb.NewMemoryDatabase()
+		backend = &testBackend{db: db}
+		api     = NewPublicFilterAPI(backend, false, deadline, false)
+		votes   = []*types.VoteEnvelope{
+			&types.VoteEnvelope{
+				VoteAddress: types.BLSPublicKey{},
+				Signature:   types.BLSSignature{},
+				Data: &types.VoteData{
+					TargetNumber: uint64(1),
```

Comment:
Add SourceNumber and Source Hash, otherwise you can't pass CI jobs.

Reply:
Fixed